Tests/GitHubRepositories.tests.ps1

```diff
@@ -14,7 +14,105 @@ $moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
 
 try
 {
-    Describe 'Modifying repositories' {
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        defaultRepoName = ([Guid]::NewGuid().Guid)
+        defaultRepoDesc = "This is a description."
+        defaultRepoHomePage = "https://www.microsoft.com/"
+        defaultRepoTopic = "microsoft"
+        modifiedRepoDesc = "This is a modified description."
+    }.GetEnumerator() | ForEach-Object {
+        Set-Variable -Force -Scope Script -Option ReadOnly -Visibility Private -Name $_.Key -Value $_.Value
+    }
+
+    Describe 'Getting repositories' {
+
+        Context -Name 'For getting a repository' -Fixture {
+            BeforeAll {
+                $repo = New-GitHubRepository -RepositoryName $defaultRepoName -Description $defaultRepoDesc -AutoInit
+            }
+            AfterAll {
+                Remove-GitHubRepository -Uri "$($repo.svn_url)" -Verbose
```

Comment:
I'm not quite sure when this practice started sneaking in, but there's no need to make this verbose.   Can you remove the `-Verbose` from here and your other test code?

```suggestion
                Remove-GitHubRepository -Uri "$($repo.svn_url)"
```

Reply:
Removed all -Verbose